.github/workflows/verify_base_branch.yaml

```
@@ -0,0 +1,40 @@
+name: Pull Request base branch verification
```

```suggestion
name: PR check
```

And possibly rename the file. I think that it's best to group "PR check" under a single file & name.